src/CrudTrait.php

```diff
@@ -17,7 +17,8 @@
     public static function getPossibleEnumValues($field_name)
     {
         $instance = new static(); // create an instance of the model to be able to get the table name
-        $type = DB::select(DB::raw('SHOW COLUMNS FROM `'.Config::get('database.connections.'.env('DB_CONNECTION').'.prefix').$instance->getTable().'` WHERE Field = "'.$field_name.'"'))[0]->Type;
+        $connectionName = $instance->getConnectionName();
+        $type = DB::connection($connectionName)->select(DB::raw('SHOW COLUMNS FROM `'.Config::get('database.connections.'.$connectionName.'.prefix').$instance->getTable().'` WHERE Field = "'.$field_name.'"'))[0]->Type;
```

Comment:
https://laravel.com/docs/5.4/helpers#method-env -- gets the value of an environment variable which technically means that some malicious entity could hijack a non-cached configuration (I think a real environment variable wins out over even .env variables).

I suppose both of these cases shouldn't occur _too often_ but it is possible.